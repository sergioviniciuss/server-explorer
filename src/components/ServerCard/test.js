import React from "react";
import { render, screen } from "@testing-library/react";
import { ServerCard } from ".";

describe("ServerCard", () => {
  it("renders with name and distance", () => {
    const { container } = render(<ServerCard name="Germany #01" distance={123} />);

    expect(screen.getByText(/Germany #01/i)).toBeInTheDocument();
    expect(screen.getByText(/123/i)).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
