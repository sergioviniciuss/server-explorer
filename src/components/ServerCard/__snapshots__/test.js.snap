// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServerCard renders ServerCard component with name and distance 1`] = `
<div
  class="wrapper"
>
  <div
    class="container"
  >
    <img
      class="image"
      src="/file"
    />
    <div
      class="info"
    >
      <div
        class="name"
      >
        Germany #01
      </div>
      <div
        class="distance"
      >
        123
      </div>
    </div>
  </div>
</div>
`;
